backend/typescript/models/camper.model.ts

```diff
@@ -101,7 +101,7 @@ const CamperSchema: Schema = new Schema({
     required: true,
   },
   chargeId: {
-    type: String,
+    type: Number,
```

Comment:
Do we want to leave this as a string

Reply:
I think this is because of the merge conflict issue. It should be string